command/build.rs

```diff
@@ -0,0 +1,34 @@
+use prost_build;
+
+// fn main() {
+//     prost_build::compile_protos(&["src/protocommand.proto"], &["src/"]).unwrap();
+// }
+
+pub fn main() {
+    let mut config = prost_build::Config::new();
+
+    config.btree_map(["."]);
+    config.message_attribute(".", "#[derive(Hash, Eq, Ord, PartialOrd)]");
+    config.type_attribute(".", "#[derive(::serde::Serialize, ::serde::Deserialize)]");
+    config.format(true);
+
+    // serialize all enums in screaming snake case
+    config.enum_attribute(".", "#[serde(rename_all = \"SCREAMING_SNAKE_CASE\")]");
+    config.enum_attribute("Order", "#[derive(Hash, Eq, Ord, PartialOrd)]");
+    config.enum_attribute("inner", "#[derive(Hash, Eq, Ord, PartialOrd)]");
+    // config.enum_attribute("worker_order.Inner", "#[derive(Hash, Eq)]");
+
+    // config.enum_attribute(
+    //     "TlsVersion",
+    //     "#[serde(deserialize_with = \"certificate::TlsVersion::from_str\")]",
+    // );
+
+    tonic_build::configure()
```

Comment:
we should not use tonic-build, but only prost-build here :)

Reply:
Cool, it works without tonic-build and tonic alltogether! `Cargo.lock` just became much lighter!